cowtown2023/src/main/java/com/team2357/frc2023/Robot.java

```diff
@@ -84,7 +74,10 @@ public void robotPeriodic() {
   /** This function is called once each time the robot enters Disabled mode. */
   @Override
   public void disabledInit() {
-    shooter.stopShooter();
+    // Wait 5 seconds then set the drive motors to coast (to allow the robot to be pushed)
+    long disabledStart = System.currentTimeMillis();
+    while (System.currentTimeMillis() - disabledStart != Constants.SWERVE.DISABLED_WAIT_TO_BREAK_MILLIS);
```

Comment:
Long-term, we'll want to use a TimerTask for this instead of blocking the main thread.